docs/en/Feature.md

```diff
@@ -0,0 +1,86 @@
+
+- retryTimesWhenSendFailed: Synchronous message retry times when send failed, default value is 2, so the producer will try to send retryTimesWhenSendFailed + 1 times at most. To ensure that the message is not lost, producer will try sending the message to another broker instead of selecting the broker that failed last time. An exception will be thrown if it reaches the retry limit, and the client should guarantee that the message will not be lost. Messages will resend when RemotingException, MQClientException, and partial MQBrokerException occur.
+- retryTimesWhenSendAsyncFailed: Asynchronous message retry times when send failed, asynchronous retry sends message to the same broker instead of selecting another one and does not guarantee that the message wont lost. 
+- retryAnotherBrokerWhenNotStoreOK: Message flush disk (master or slave) timeout or slave not available (return status is not SEND_OK), whether to try to send to another broker, default value is false. Very important messages can set to true.
+
+## Flow Control
+Producer flow control, because broker processing capacity reaches a bottleneck; Consumer flow control, because the consumption capacity reaches a bottleneck.
+
+Producer flow control:
+- CommitLog file locked time exceeds osPageCacheBusyTimeOutMills, parameter default value is 1000 ms, return flow control.
+- If transientStorePoolEnable == true, and the broker is asynchronous flush disk type, and resources are insufficient in the transientStorePool, reject the current send request and return flow control.
+- The broker checks the head request wait time of the send request queue every 10ms. If the wait time exceeds waitTimeMillsInSendQueue, which default value is 200ms, the current send request is rejected and the flow control is returned.
+- The broker implements flow control by rejecting send requests.
+
+Consumer flow control:
+- When consumer local cache messages number exceeds pullThresholdForQueue, default value is 1000. 
+- When consumer local cache messages size exceeds pullThresholdSizeForQueue, default value is 100MB. 
+- When consumer local cache messages span exceeds consumeConcurrentlyMaxSpan, default value is 2000.
+
+The result of consumer flow control is to reduce the pull frequency.
+
+## Dead Letter Queue
+Dead letter queue is used to deal messages that cannot be consumed normally. When the first consumption of a message fails, the message queue will automatically resend the message. If the consumption still fails after the maximum number retry, it indicates that the consumer cannot properly consume the message under normal circumstances. At this time, the message queue will not immediately abandon the message, but send it to the special queue corresponding to the consumer.
```

Comment:
i think it is better to change "When the first consumption of a message fails," as
When a message is consumed failed at first time,